pkgs/applications/networking/cluster/helm/plugins/default.nix

```diff
@@ -0,0 +1,36 @@
+{ stdenv, fetchFromGitHub, fetchurl, fetchzip }:
+let
+
+  buildHelmPlugin = { name, ... }@attrs:
```

Comment:
Can we expose this as well? So that people can build their own helm plugins in their own overlays?